config/development.yaml

```diff
@@ -28,3 +28,11 @@ storage:
     default_segment_number: 2
 
   handle_collection_load_errors: true
+
+
+tls:
+  # Certificate chain file
+  cert: ./tests/tls/cert/cert.pem
+
+  # Private key file
+  key: ./tests/tls/cert/key.pem
```

Comment:
Do we just put this here to ensure we can parse `TlsConfig` without `ca_cert` defined?

Reply:
It was a debug change, forgot to roll back